src/libAtomVM/opcodesswitch.h

```diff
@@ -1538,7 +1538,9 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
 
         ctx->cp = module_address(mod->module_index, mod->end_instruction_ii);
         ctx->saved_ip = mod->labels[label];
+        #ifndef DOXYGEN_SKIP_SECTION /* documented in scheduler.h */
```

Comment:
this feels so weird. Why are we conditionally disabling a piece of code according to some documentation thing?

Reply:
No, this is conditionally disabling documentation, because this code confuses doxygen.

Reply:
We already do this in at least one other place:
https://github.com/atomvm/AtomVM/blob/f26a64391e3d7af67ddd80cfa0b949e3ecb65bb4/src/libAtomVM/listeners.h#L147C1-L153C34

Reply:
I find it really ugly, but ok, let's keep using it.

Reply:
I agree. I was unhappy with this solution from the Doxygen documentation, and searched for alternatives, but any others are even more invasive.

It would be fantastic if there were a plugin for ex_doc that did doxygen like parsing of C api documentation, this could be helpful for documenting C APIs for nifs and ports in many projects.